### ISSUE TITLE
Issue with single default parameter

### ISSUE THREAD

**Issue by objeck** [objeck/objeck-lang#35]
The following code did not compile.
```
class Test {
	function : Main(args : String[]) ~ Nil {
		Foo();
		Foo(false);
	}

	function : Foo(a : Bool := false) ~ Nil {		
		a->PrintLine();
	}
}
```

**Comment by objeck** [comment]
Issue fixed in commit c5d33956230cac9798c2cd531045de5c215dda04

**Comment by objeck** [comment]
fixed in 5.9.7
